src/services/mongoose/phone-code.ts

```diff
@@ -0,0 +1,54 @@
+import { VALIDITY_TIME_CODE } from "@config/app"
```

Comment:
Values like this are critical business logic and should either be defined in domain - or if they are configurable then injected from the app layer - see below.